### DISCUSSION TITLE
SignalR Client (.net 8) missing assembly (IStatefulReconnectFeature)

### DISCUSSION THREAD

**Discussion by carlberg74** [dotnet/aspnetcore#52166] [General]
Created a new project with ChatHub example and added SignalR.Client, then suddely I get exception:

System.TypeLoadException: Could not load type 'Microsoft.AspNetCore.Connections.Abstractions.IStatefulReconnectFeature' from assembly 'Microsoft.AspNetCore.Connections.Abstractions, Version=8.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'.

```
.csproj:
<Project Sdk="Microsoft.NET.Sdk.Web">
    <PropertyGroup>
        <TargetFramework>net8.0</TargetFramework>
        <Nullable>enable</Nullable>
        <ImplicitUsings>enable</ImplicitUsings>
        <EnablePreviewFeatures>true</EnablePreviewFeatures>
    </PropertyGroup>
    <ItemGroup>
      <PackageReference Include="Microsoft.AspNetCore.Connections.Abstractions" Version="8.0.0" />
      <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="8.0.0" />
    </ItemGroup>
</Project>

```
As soon as I try to StartAsync using HubConnectionBuilder, I get the exception..

```
var  hubConnection = new HubConnectionBuilder()
       .WithUrl(url, options =>{  })
       .Build();
       await hubConnection.StartAsync();
```

Dont know why I get the error in the first place, but I cant even get around it with enabling EnablePreviewFeatures...
Have I enabled preview features wrong? 
I can find the interface in the Microsoft.AspNetCore.Connections.Abstractions and yes, its marked as preview feature..
but..  

Anyone else seen this or have an ide why this occurs..

**Comment by BrennanConroy** [comment]
What does `dotnet --info` show? I just tried this and it doesn't repro.

**Comment by carlberg74** [comment]
dotnet --info
.NET SDK:
 Version:   8.0.100-preview.7.23376.3
 Commit:    daebeea8ea


Did not realize I was running a preview..    downloading v8.0.100 release...

**Reply by carlberg74** [reply]
...and suddenly things works again.. thank you..
